zig-dxread/src/main.zig

```diff
@@ -0,0 +1,72 @@
+const std = @import("std");
+const fs = std.fs;
+const gpa = std.heap.GeneralPurposeAllocator(.{}){};
+const allocator = gpa.allocator;
+const ArrayList = std.ArrayList;
+const erorset_1 = error{OutOfMemory};
+
+const RECORD_SIZE = 80 * 384;
+
+const DX_header = struct {
```

Comment:
structs should be `TitleCase`, so this would be `DXHeader`

Reply:
thanks!